BERT/main.py

```diff
@@ -157,12 +197,27 @@ def run_main(args):
     #     WorkerModule("worker2", pipeline_on_devices(1, 3, 5, 7, n_encoders=args.nlayers // 2, include_head=True), MLMTaskSharder, ntokens, args.emsize, args.nhead, args.nhid, args.dropout),
     # )
 
-    model = DistributedCPURPCSequential(
-        WorkerModule("worker1", pipeline_on_devices(5, 2, include_embeddings=True, n_encoders=args.nlayers // 6, n_encoders_on_last_gpu=args.nlayers // 6), MLMTaskSharder, ntokens, args.emsize, args.nhead, args.nhid, args.dropout),
-        WorkerModule("worker2", pipeline_on_devices(0, 7, n_encoders=args.nlayers // 3), MLMTaskSharder, ntokens, args.emsize, args.nhead, args.nhid, args.dropout),
-        WorkerModule("worker3", pipeline_on_devices(1, 4, n_encoders=args.nlayers // 3), MLMTaskSharder, ntokens, args.emsize, args.nhead, args.nhid, args.dropout),
-        WorkerModule("worker4", pipeline_on_devices(6, 3, n_encoders=args.nlayers // 6, n_encoders_on_first_gpu=args.nlayers // 6, include_head=True), MLMTaskSharder, ntokens, args.emsize, args.nhead, args.nhid, args.dropout),
-    )
+    layers = [RemoteModule("w0/cuda:0", MLMTaskEmbedding, (ntokens, args.emsize))]
```

Comment:
Instead of `RemoteModule("w0/cuda:0", MLMTaskEmbedding, (ntokens, args.emsize))`
Do:
MLMTaskEmbedding(ntokens, args.emsize).to('cuda:0')